src/utils/shared.py

```diff
@@ -89,4 +89,4 @@ def set_order(objects, order_attr_name, pk_list):
 
   
 def day_month(date):
-    return date.strftime("%d-%m")
+    return date.strftime("%d-%M")
```

Comment:
Should be `%b`

Reply:
For some reason %b is returning with a capitalised month eg `Jan` and when using `%b` in the template it correctly returns a lowercase month eg `jan` so to hamonise I've set them both to `%M` which according to Django docs returns:

`M | Month, textual, 3 letters. | 'Jan'`

Reply:
`stfrtime` is a python `datetime` function and `%M` renders minutes, not the month in textual form.

Reply:
🤣